Sources/Clock.swift

```diff
@@ -75,12 +79,20 @@ public struct Clock {
     }
 
     private static func loadFromDefaults() {
-        guard let stored = UserDefaults.standard.value(forKey: kDefaultsKey) as? [String: TimeInterval],
+        guard let stored = self.defaults.value(forKey: kDefaultsKey) as? [String: TimeInterval],
             let previousStableTime = TimeFreeze(from: stored) else
         {
             self.stableTime = nil
             return
         }
         self.stableTime = previousStableTime
     }
+
+    private static var defaults: UserDefaults {
```

Comment:
maybe it's worth to keep this as a state so we don't create an instance of `UserDefaults` on each call?

Reply:
I extracted all the `UserDefaults` stuff to a separate type (mostly so that I could test it)